### DISCUSSION TITLE
solar radiation annual radiation threshold

### DISCUSSION THREAD

**Discussion by RicardoGomesIST** [architecture-building-systems/CityEnergyAnalyst#3170] [Q&A] [answered]
Hi, 
I have a question about the solar radiation annual radiation threshold. Does it represent the global horizontal radiation or consider already the inclination of the solar pannel (asked on the panel-tilt-angle)? Or other...?
TY

**Comment by shanshanhsieh** [comment, marked as answer]
Hi @RicardoGomesIST, 
the threshold does not consider the panel inclination.
- for sensors on the rooftop, it refers to global horizontal radiation.
- for sensors on the facade, it refers to the radiation received on the vertical surface.
hope this helps.
